src/pages/editlist/EditListPage.tsx

```diff
@@ -81,7 +81,9 @@ export const EditListPage:FC = () => {
         if (isListBecameActive) {
           showSuccessMessage({ message: t('callout.list.active', {
             listName: state[FIELD_NAMES.LIST_NAME]
-          }) });
+          }),
+          // Auto-removing does not work if messages appears in same time and has same timout
+          timeout: 5999 });
```

Comment:
Let's add these to global constants so we ensure we have a different number for each message type.

Reply:
It is only one place where we have this error. I suggest to do that in case if we will face it somewhere else